TicTacToe2/TicTacToeGame.cs

```diff
@@ -12,8 +12,9 @@ public void Play()
             while (!winner)
             {
                 DrawBoard();
-                UserInput();
-                winner = CheckWinDraw();
+                GetUserInput();
+                CheckForDraw();
```

Comment:
Checking for win and draw have now been split into separate functions, but the game loop is not affected by a draw. Draw should be included in the loop condition in addition to winner